### PR TITLE
Update dev-dependencies

### PR DESCRIPTION
**Pull request by piatrashkakanstantinass**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,12 +8,16 @@ documentation = "https://github.com/bee-san/pyWhat/wiki"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```
Comment (bee-san) [inline pyproject.toml:11]: What happens between .6 and .6.2 that we need this? :)

Comment (piatrashkakanstantinass) [reply pyproject.toml:11]: black requires python 3.6.2 or higher @bee-san

Comment (SkeletalDemise) [reply pyproject.toml:11]: Then add it under the dev dependencies so that people can still use Python `3.6.0` if they want.

Comment (bee-san) [reply pyproject.toml:11]: Yeah I agree with Skeletal here, REMnux requires 3.6 :shrug: :(

**Review by bee-san — APPROVED** [review]

**Comment by bee-san** [comment]
merge conflicts :(

**Review by SkeletalDemise — APPROVED** [review]

**Pull request merged**